project/Dependencies.scala

```diff
@@ -5,10 +5,11 @@ object Dependencies {
 
   // Version numbers
 
-  val PlayVersion = "2.6.5" // if you update this, you probably need to update the following two
-  val PlayJsonVersion = "2.6.5"
-  val PlayStandaloneWsVersion = "1.1.0"
-  val PlayFileWatchVersion = "1.1.0"
+  val PlayVersion = "2.6.6" // if you update this, you probably need to update the following two
+  val PlayJsonVersion = "2.6.6"
+  val PlayStandaloneWsVersion = "1.1.2"
+  val TwirlVersion = "1.3.12"
+  val PlayFileWatchVersion = "1.0.0"
```

Comment:
This seems a step back (from `1.1.0` to `1.0.0`), that's correct?

Reply:
Yes, for now. I changed my mind about forcing the upgrade in Lagom, since we are already using the new watcher for sbt 1.0 users in Play. We can decide once we're sure about the new version's stability. (People have reported issues that may or may not be related to the watch service.)